src/collections/_documentation/platforms/javascript/sourcemaps/troubleshooting.md

```diff
@@ -34,21 +40,46 @@ Sentry will look for a matching artifact named exactly `https://example.com/dist
 
 Note also that Sentry will resolve relative paths. For example, if you have the following:
 
-```JavaScript
+```javascript
 // -- end app.min.js (located at https://example.com/dist/js/app.min.js)
 //# sourceMappingURL=app.min.js.map
 ```
 
 Sentry will resolve `sourceMappingURL` relative to `https://example.com/dist/js/` (the root path from which `app.min.js` was served). You will again need to name your source map with the full URL: `https://example.com/dist/js/app.min.js.map`.
 
-If you serve the same assets from multiple origins, you can also alternatively use our tilde (~) path prefix to ignore matching against protocol + hostname. In which case, `~/dist/js/app.min.js.map`, will also work. See: [Assets Accessible at Multiple Origins](#assets-multiple-origins).
+If you serve the same assets from multiple origins, you can also alternatively use our tilde (~) path prefix to ignore matching against protocol + hostname. In which case, `~/dist/js/app.min.js.map`, will also work. See: [Assets Accessible at Multiple Origins]({%- link _documentation/platforms/javascript/sourcemaps/availability.md -%}#uploading-source-maps-to-sentry).
```

Comment:
"If you serve the same assets from multiple origins, you can also alternatively use our tilde (~) path prefix to ignore matching against protocol + hostname. In which case, `~/dist/js/app.min.js.map`, will also work."

Second sentence is a fragment, so let's combine them:

"If you serve the same assets from multiple origins, you can also alternatively use our tilde (~) path prefix to ignore matching against protocol + hostname (in which case `~/dist/js/app.min.js.map` will also work).